ai_ta_backend/nomic_logging.py

```diff
@@ -115,7 +115,7 @@ def log_convo_to_nomic(course_name: str, conversation) -> str:
       }]
 
       # create embeddings
-      embeddings_model = OpenAIEmbeddings() # type: ignore
+      embeddings_model = OpenAIEmbeddings(openai_api_type="azure") # type: ignore

```

Comment:
@star-nox Just curious, are we going to have to change this back to OpenAI when our Azure credits expire at the end of January? Thanks.

Reply:
Thanks for pointing this out! I have created a global variable for this now so we don't have to change it back everywhere when the credits expire.